wp-content/themes/access/single-programs.php

```diff
@@ -111,50 +111,57 @@
   array_push($context['schema'], $special_announcement);
 }
 
-if($program->validateFaq() === true) {
+/**
+* The array $questions has a set of elements that are the questions to be added
+* to the $faq variable which will be added to the schema as the `FAQPage`
+* section.
+*/
+$questions = [
+  array(
+    '@type' => 'Question',
+    'name' => "How does $program->program_name work?",
+    'acceptedAnswer' => array(
+      '@type' => 'Answer',
+      'text' => $program->faqAnswer('field_58912c1a8a81b')
+    )
+  ),
+  array(
+    '@type' => 'Question',
+    'name' => "Am I eligible for $program->program_name?",
+    'acceptedAnswer' => array(
+      '@type' => 'Answer',
+      'text' => $program->faqAnswer('field_58912c1a8a82d')
+    )
+  ),
+  array(
+    '@type' => 'Question',
+    'name' => "What do I need in order to apply to $program->program_name?",
+    'acceptedAnswer' => array(
+      '@type' => 'Answer',
+      'text' => $program->faqAnswer('field_589de18fca4e0')
+    )
+  ),
+  array(
+    '@type' => 'Question',
+    'name' => "How do I Apply to $program->program_name?",
+    'acceptedAnswer' => array(
+      '@type' => 'Answer',
+      'text' => join('', [$program->faqAnswer('field_58912c1a8a850'),
+                          $program->faqAnswer('field_58912c1a8a885'),
+                          $program->faqAnswer('field_58912c1a8a900'),
+                          $program->faqAnswer('field_58912c1a8a8cb')])
+    )
+  )
+];
+
+if($program->addQuestionsToSchemaFaq($questions)) {
```

Comment:
`if($` > `if ($`

Reply:
I would just move all of this into a single method in the program controller called `getFaqSchema` then you can shrink this down a bit to a one-liner;

```
array_push($context['schema'], $program->getFaqSchema());
```